crates/uv/tests/tool_upgrade.rs

```diff
@@ -358,14 +330,9 @@ fn test_tool_upgrade_constraint() {
 
     ----- stderr -----
     warning: `uv tool upgrade` is experimental and may change without warning
-    Resolved [N] packages in [TIME]
-    Prepared [N] packages in [TIME]
-    Uninstalled [N] packages in [TIME]
-    Installed [N] packages in [TIME]
-     - babel==2.6.0
-     + babel==2.13.1
-     - pytz==2018.5
-     + setuptools==69.2.0
+    Updated babel v2.6.0 -> v2.13.1
+    Removed pytz v2018.5
+    Added setuptools v69.2.0
```

Comment:
Unfortunately I find this a little confusing. At first, I thought the other things being modified were _other_ tools. I think what I'd expected to see is

```
Updated babel v2.6.0 -> v2.13.1
- babel==2.6.0
+ babel==2.13.1
- pytz==2018.5
+ setuptools==69.2.0
```

Or maybe some other way of nesting the environment modifications separately from the main tool itself? This would help in the `upgrade --all` case as well.

Reply:
And if the tool itself isn't updated?

Reply:
(It's fine, it's very easy to change.)

Reply:
Ah I forgot about dependency updates. That does make it a little trickier to summarize.

I guess the summary line could be "Updated babel environment"? My only concern is that they may be confused as to why the tool itself wasn't updated. Seems verbose to clarify though.

I also considered ""Updated babel dependencies" but this includes `--with` so that's not quite correct.

Reply:
Sounds good. I think your format is good and neatly solves for multiple tool updates.